test/functional/shared/resources/gateway_test.go

```diff
@@ -441,7 +439,7 @@ func testGatewayWithPortForward(t *testing.T, ctx context.Context, at shared.RPT
 		t.Logf("Portforward session active at %s", baseURL)
 
 		for _, test := range tests {
-			if err := testGatewayAvailability(hostname, baseURL, test.Path, test.ExpectedStatusCode, isHttps); err != nil {
+			if err := testGatewayAvailability(t, hostname, baseURL, test.Path, test.ExpectedStatusCode, isHttps); err != nil {
```

Comment:
As a style question/comment: is it a best practice to change existing arity order when adding a new param? In this case the function isn't public, but it's a bad habit because this would break usage of the function outside the file.

Reply:
in general I would add the new param to the end, but in this case, it seems to be go convention to put the `t *testing.T` param at the beginning of the param list

Reply:
or at least it's pretty heavily used in our project